ros/angel_system_nodes/angel_system_nodes/object_detector.py

```diff
@@ -48,9 +49,18 @@ def __init__(self):
             1
         )
 
+        # Start a service to allow other nodes to get the current image size
+        self._image_size_service = self.create_service(QueryImageSize,
```

Comment:
Would it be better if this service was defined / provided by the image converter node `ros/angel_datahub/src/image_converter.cpp`? That is the "source" (that we control at least) of the image data that is being emitted into the system.

Reply:
Just looping myself into this conversation

Reply:
Went ahead and moved the service there.